src/controllers/user.controller.js

```diff
@@ -0,0 +1,15 @@
+const {getAll} = require('../db/queries/user/read');
+const authMiddleware = require('../middlewares/authentication.middleware');
+
+module.exports = {
+    getAllUsers(req, res) {
+        const auth = new authMiddleware(req, res);
+
+        if (auth.checkToken()) {
```

Comment:
`auth.checkToken()` doesn't show that our token is valid.
We need use `auth.getUser()` as a variable before functions to check the token.